packages/protocol/contracts/governance/SlasherUtil.sol

```diff
@@ -65,11 +66,16 @@ contract SlasherUtil is Ownable, Initializable, UsingRegistry, UsingPrecompiles
     uint256[] memory groupElectionIndices
   ) internal {
     ILockedGold lockedGold = getLockedGold();
+    // Cannot set reward as local because of solc stack too deep
+    uint256 penalty = Math.min(
```

Comment:
Instead, what if in `lockedGold.slash`, slash `actualPenalty = Math.min(getAccountTotalLockedGold(), penalty)`, and `actualReward = Math.min(reward, actualReward)`?